.gitattributes

```diff
@@ -0,0 +1,16 @@
+* text=auto
```

Comment:
I have never had to specify line-endings, ever, in any text editor I use, for any file I have created in any of Basho's repos. Again, this is because I have `git` configured correctly.

If I create new `.rb` files on Windows, they are `CRLF` ending, on Linux, they are `LF`. Commiting them does not cause issues, nor does checking them out again.

The Java client doesn't need this file either, to be honest.

If anything, we should add something to all of our `README.md` files suggesting that GitHub's `git` configuration be used.